plugins/spi/SPIOutput.h

```diff
@@ -178,6 +183,9 @@ class SPIOutput: public ola::rdm::DiscoverableRDMControllerInterface {
   static const uint16_t LPD8806_SLOTS_PER_PIXEL;
   static const uint16_t P9813_SLOTS_PER_PIXEL;
   static const uint16_t P9813_SPI_BYTES_PER_PIXEL;
+  static const uint16_t APA102_SLOTS_PER_PIXEL;
+  static const uint16_t APA102_SPI_BYTES_PER_PIXEL;
+  static const uint8_t  APA102_START_FRAME_BYTES;
```

Comment:
You may as well make this a uint16_t to match the others (although I'm not entirely sure why they were in the first place).